blog/2022-12-19-openhab-3-4-release.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: BlogPost
+category: blog
+title: openHAB 3.4 Release
+author: Kai Kreuzer
+date: '2022-12-19T14:38:57+01:00'
+previewimage: /uploads/winter22.jpg
+tags:
+  - news
+excerpt: >-
+  As every year before the festive season, we are happy to announce another  
+  stable release of openHAB that brings many additions, improvements and fixes!
+---
+openHAB continues to steadily evolve - since our summer release 3.3 the community continued contributing with undiminished speed, which has led to openHAB 3.4 being the best openHAB release ever.
+
+You can find the full list of new add-ons, enhancements and fixes in our [**official release notes**](https://github.com/openhab/openhab-distro/releases/tag/3.4.0).
+
+With that being said, we again want to share some statistics that show the progress in numbers.
+
+# Activity
+
+The new release brings 17 new add-ons from a wide range of applications - starting from another important vehicle integration with [MercedesMe](https://next.openhab.org/addons/bindings/mercedesme/) to smart gadgets like the connected meat thermometer [Meater](https://www.openhab.org/addons/bindings/meater/). In these 6 months of development, the add-ons repo counted 619 pull requests, resulting in more than 100,000 new lines of code.
```

Comment:
Is the `next.openhab.org` link correct here for Mercedes Me?

Reply:
No, thanks for catching it.